http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/GalaxyAppConfig.scala

```diff
@@ -1,6 +1,11 @@
 package org.broadinstitute.dsde.workbench.leonardo.config
 
 import org.broadinstitute.dsde.workbench.google2.KubernetesSerializableName.NamespaceName
-import org.broadinstitute.dsde.workbench.leonardo.{ReleaseName, ServiceConfig}
+import org.broadinstitute.dsde.workbench.leonardo.{ChartName, KubernetesServiceAccount, ReleaseName, ServiceConfig}
 
-case class GalaxyAppConfig(releaseName: ReleaseName, namespaceNameSuffix: NamespaceName, services: List[ServiceConfig])
+case class GalaxyAppConfig(releaseNameSuffix: ReleaseName,
+                           chart: ChartName,
+                           namespaceNameSuffix: NamespaceName,
+                           services: List[ServiceConfig],
+                           serviceAccountSuffix: KubernetesServiceAccount,
```

Comment:
is this just suffix or a complete ksa?

Reply:
Just a suffix - I'm generating the ksa as `${appName}-${serviceAccountSuffix}`

Reply:
Maybe the suffixes should be `String`s?

Reply:
I think `String` sounds good to me...there's no other `String` param in this data type anyway, so there's no chance we'll mix it up with another param